samples/wifi/raw_tx_packet/Kconfig

```diff
@@ -115,4 +115,29 @@ config RAW_TX_PACKET_SAMPLE_QUEUE_NUM
 	  3 - Voice
 	  4 - Beacon.
 
+choice
+	prompt "Select transmission mode"
+	default RAW_TX_PACKET_SAMPLE_TRANSMISSION_MODE_CONTINUOUS
+
+config RAW_TX_PACKET_SAMPLE_TRANSMISSION_MODE_CONTINUOUS
+	bool "Continuous transmission mode"
+
+config RAW_TX_PACKET_SAMPLE_TRANSMISSION_MODE_FIXED
+	bool "Fixed transmission mode"
+
+config RAW_TX_PACKET_SAMPLE_FIXED_NUM_PACKETS
```

Comment:
move this out of the choice and then protect with `if RAW_TX_PACKET_SAMPLE_TRANSMISSION_MODE_FIXED` instead of depends on.